renderdoc/driver/gl/gl_common.h

```diff
@@ -56,11 +56,17 @@ struct GLWindowingData
 };
 
 #elif ENABLED(RDOC_LINUX)
+
+#ifdef RENDERDOC_SUPPORT_GL
```

Comment:
How will this work if both GL and GLES are enabled? It seems like that's possible on linux.

They're not mutually exclusive just that android doesn't support desktop GL obviously. I think we want to make these independent #ifs to include both sets of headers (assuming there are no conflicts).